cdserver/app/models/shelter_date.rb

```diff
@@ -0,0 +1,19 @@
+class ShelterDate
+  include Comparable
+  attr_reader :date, :cutoff
+
+  def initialize(datetime, cutoff)
+    @cutoff = cutoff
```

Comment:
Looking again this morning, maybe we can keep it as number of hours for now.    Simplicity may be better, since we can easily make it more granular later if we need to.   I guess my only point would be to document what it really is somehow.  I believe it is an integer that represents the number of hours past midnight.  Either make that the method name or parameter name or add a comment to that effect, and I think we are good.